index.html

```diff
@@ -19,34 +19,78 @@
         <li class="logo-link"><a href="./index.html">QUIZ NAME</a></li>
       </ul>
       <ul class="navigation-list">
-        <li><a href="/pages/actors.html">Actors</a></li>
-        <li><a href="/pages/frames.html">Movie Frames</a></li>
-        <li><a href="/pages/sounds.html">Soundtracks</a></li>
+        <li>
+          <button href="/pages/actors.html" class="js-quiz-target actors-click" data-target="actors">Actors</button>
+        </li>
+        <li>
+          <button href="/pages/frames.html" class="js-quiz-target frames-click" data-target="frames">
+            Movie Frames
+          </button>
+        </li>
+        <li>
+          <button href="/pages/sounds.html" class="js-quiz-target soundtracks-click" data-target="soundtracks">
+            Soundtracks
+          </button>
+        </li>
       </ul>
       <div class="burger">
         <div class="line1"></div>
         <div class="line2"></div>
         <div class="line3"></div>
       </div>
     </nav>
-    <main>
-      <div class="home-container">
-        <div class="text-content">
-          <h1>Welcome to the MOVIE QUIZ</h1>
-          <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
-            dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex
-            ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat
-            nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa
-          </p>
-        </div>
-        <div class="buttons-container">
-          <button><a href="/pages/actors.html">Actors</a></button>
-          <button><a href="/pages/frames.html">Movie Frames</a></button>
-          <button><a href="/pages/sounds.html">Soundtracks</a></button>
-        </div>
+
+    <!-- <template id="homeTemplate"> -->
+    <div class="home-container">
+      <div class="text-content">
+        <h1>Welcome to the MOVIE QUIZ</h1>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+          magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+          consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+          pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa
+        </p>
+      </div>
+      <div class="buttons-container">
+        <button><a href="/pages/actors.html">Actors</a></button>
+        <button><a href="/pages/frames.html">Movie Frames</a></button>
+        <button><a href="/pages/sounds.html">Soundtracks</a></button>
+      </div>
+    </div>
+    <!-- </template> -->
+
+    <!-- <template id="quizTemplate"> -->
+    <div id="quiz" class="quiz-container hidden">
```

Comment:
Istnieje atrybut `hidden`, który ukrywa elementy na stronie, nie trzeba do tego klasy: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden

Wtedy w Jsie można to robić tak:
```
document.getElementyById('quiz').hidden = true;
```
albo false ;-)